README.md

```diff
@@ -1,22 +1,103 @@
 # Celeste on iOS
 
-## what you'll need
-- an **FNA** build of Celeste (this repo has been tested with the Steam version only, the itch.io version *should* work too)
-- MacOS (if you don't have access to a MacOS machine, you can use a MacOS VM)
-- XCode
-- Visual Studio for Mac with the C#, .NET and Xamarin.iOS workloads
-- patience
-
-## how to build
-- install the needed software mentioned above
-- run `./build.sh`
-- follow the instructions
-- hope it works :3
-
-these instructions might be slightly wrong or have missing information because i wrote this like 3 months ago and don't remember every detail - if you have issues feel free to open an issue :3
-
-## installing
-the script will tell you where the final .ipa file is located - just sideload it to your iOS device using something like TrollStore, AltStore or similar
+# How to Build
+
+This guide will walk you through building a Celeste IPA for iOS using macOS. Please note that **only macOS** is supported for this process. If you don't have a Mac you'll have to use a Virtual Machine.
+
+## Prerequisites
+
+1. **Install Xcode**  
+   - Make sure you have Xcode installed and set up correctly. You can download it from the [Mac App Store](https://apps.apple.com/us/app/xcode/id497799835).
+   - Install the iOS Simulator in Xcode
+
+2. **Install Git**  
+   If Git isn't already installed, running `git` from your terminal will often prompt to install it.
+
+3. **Install FMOD Engine**  
+   - Go to the FMOD download page: [FMOD v1.10.09](https://www.fmod.com/download?version=1.10.09#modengine).  
+   - You will need to create a free FMOD account to download.
+   - Make sure that the FMOD version is set to **v1.10.09 (unsupported)**
+   - Download the **iOS** version of **FMOD Engine**
+   - After downloading, mount the downloaded `fmodstudioapi11009ios-installer.dmg` file by double-clicking it. **No need to do anything further**; simply ensure it's mounted.
+
+4. **Install Visual Studio for macOS**  
+   - Download Visual Studio for macOS from [here](https://visualstudio.microsoft.com/vs/older-downloads/).
+     - If that download link doesn't work, use [this one](https://visual-studio.macupdate.com)
```

Comment:
please do not link to shady 3rd-party sites, who knows what they injected into that dmg
its *probably* fine, but i'd rather not have that stuff linked.

Reply:
Yeah the only reason I put that in was because someone I was trying to help (which is how this guide was partially made) was having problems signing into the Microsoft website for some reason, and i just thought it'd be good to have an alternate link in case that happened to someone else. I also looked into it and apparently the website is very trusted and has been around since before the app store launched on macOS, but again happy to remove the link :)

Reply:
Actually just done another search and must have missed this "MacUpdate used to be a great resource for mac apps. No longer. They were purchased by the spamware company, MacKeeper awhile back" 
Yeah I'll defo remove the link haha